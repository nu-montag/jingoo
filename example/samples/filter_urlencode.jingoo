for_qs(default): {{ "http://example.com/foo bar" | urlencode }}
for_qs(false): {{ "http://example.com/foo bar" | urlencode(for_qs=false) }}
for_qs(true): {{ "http://example.com/foo bar" | urlencode(for_qs=true) }}
