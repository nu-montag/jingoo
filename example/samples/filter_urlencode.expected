for_qs(default): http%3A//example.com/foo%20bar
for_qs(false): http%3A//example.com/foo%20bar
for_qs(true): http%3A//example.com/foo+bar
